packages/amplify-graphql-conversation-transformer/src/__tests__/__snapshots__/amplify-graphql-conversation-transformer.test.ts.snap

```diff
@@ -293,7 +295,12 @@ export function response(ctx) {
 
   return streamEvent;
 }
-"
+
+function generateRandomPadding() {
+  const base = 'abcdefghijklmnopqrstuvwxyz0123456789';
+  const rand = Math.floor(Math.random() * 36);
+  return base.slice(0, rand);
```

Comment:
Is empty-string a valid padding value?

Reply:
Yes, an empty string is a valid padding value.